### PR TITLE
Add new card variation prototype

### PR DESCRIPTION
**Pull request by AriannaChau**
## Overview

This PR adds a prototype that includes some card container variations. 
The examples are: 
- Light card on a light background
- Darker card on light background.
- Darker card on dark background
- Light card on dark background.

## Screenshots

**Cards on light background:**
Screen Shot 2021-05-26 at 12 40 40 PM

**Cards on dark background:**
Screen Shot 2021-05-26 at 12 41 31 PM

## Testing

1. [Deploy link](https://deploy-preview-1241--cloudfour-patterns.netlify.app/?path=/story/prototypes-card-variations--example)

---

- Fixes https://github.com/cloudfour/cloudfour.com-patterns/issues/1090

### CODE REVIEW

File: src/prototypes/card-variations/example/example.twig [hunk 1]
```diff
@@ -0,0 +1,100 @@
+<div id="card-variations">
+  <div class="wrapper-container light o-container o-container--prose">
+    <div class="card-one o-media o-container__content">
+      <div class="test">
+        <div class="o-media__object">
+          <div class="c-avatar">
+            <img
+              src="static/media/src/objects/media/demo/megan.png"
```
Comment (gerardo-rodriguez) [inline src/prototypes/card-variations/example/example.twig:8]: I am not 100% sure but I _think_ if the images exist in the `static/` directory you can use that path. 

For example:
```html
<img src="clouds/cloud-white-left.svg">
```
Would look for the image in `/static/clouds/cloud-white-left.svg`

So I _think_ you can add the `megan.png` file into the `/static/media/` directory then access it via `/media/megan.png`. 

I think. 😅

Comment (AriannaChau) [reply src/prototypes/card-variations/example/example.twig]: That worked!! Thank you!!

**Review by gerardo-rodriguez — COMMENTED** [review]
Overall, this LGTM!

I'll defer final approval to @aileenjeffries since she provided more specific design feedback. 😉

**Review by aileenjeffries — APPROVED** [review]
Since this is a prototype, strict review is probably not needed? I'd be interested in seeing the "footer" addition to the cards that don't have it. There might be some design problems to work out there, especially with the light gray on white bg card. Any color other than light gray for the footer might take too much attention away from the main content. Same with the lighter blue on a dark blue bg card.

**Pull request merged**